5.0.0/buster-slim/Dockerfile

```diff
@@ -34,4 +34,8 @@ WORKDIR $IDSVR_HOME
 COPY --chown=idsvr idsvr-5.0.0/idsvr /opt/idsvr
 COPY --chown=idsvr first-run /opt/idsvr/etc/first-run
 
+RUN find /opt/idsvr -type f -exec chmod a-w {} \; && \
```

Comment:
This command will be run by the `idsvr` user when building the image, which will probably lead to a failure. 
move the `USER idsvr:idsvr` instruction from line 27 below this `RUN`

Reply:
Fixed.